phpunit.bootstrap.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require_once dirname(__FILE__) . '/Cpanel/Util/Autoload.php';
+use PHPUnit\Framework\TestCase;
+
+class CpanelTestCase extends TestCase
+{
+    protected function _makeMock($className, $methods = array(), $args = array(), $mockName = '', $callConst = true, $callClone = true, $callA = true) {
```

Comment:
ah, I see `_makeMock` is a custom one for our class. Any reason to make it not private?

Reply:
If made private instead of protected, the child classes won't be able to access it.